packages/translate/lib/texts.ts

```diff
@@ -13,6 +13,7 @@ export const texts = Object.freeze({
 	'startupError.candyBoxOffline.desc': () => 'Check that the cable plugged into the CandyBox delivers Internet connection or reboot the CandyBox and allow about 5 minutes for it to initialize',
 	'startupError.suitestDriveOffline': () => 'Cannot continue: SuitestDrive controlling this device is offline',
 	'startupError.suitestDriveOffline.desc': () => 'SuitestDrive controlling this device is not currently running or is offline. Please verify that the host computer has Internet connection and that SuitestDrive is running',
+	'startupError.suitestDriveServiceOffline': () => 'The SuitestDrive process is not running on the CandyBox.',
```

Comment:
@andriichumak we have a ticket https://mineus.atlassian.net/browse/SUIT-14113 to rename CandyBox to Control unit, should here it also be "Control unit"

Reply:
Nope, in this case it's actually CandyBox